framework/src/tinyalsa/tinyalsa.c

```diff
@@ -739,6 +779,10 @@ int pcm_close(struct pcm *pcm)
 	if (pcm->fd >= 0) {
 		close(pcm->fd);
 	}
+
+	pcm->nextSize = 0;
+        pcm->nextOffset = 0;
+        pcm->nextBuf = NULL;
```

Comment:
Violations of coding rule at here and above line.

Reply:
Fixed